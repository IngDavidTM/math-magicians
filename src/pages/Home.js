import React from 'react';
import '../stylesheet/Home.css';

const Home = () => (
  <div className="home">
    <h2>Welcome to our page</h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus
      sed quia nisi quos ipsum et possimus ducimus reiciendis deserunt quasi,
      alias facilis odio sunt porro magni dolorum autem ipsa facere.Lorem ipsum
      dolor sit amet, consectetur adipisicing elit. Necessitatibus sed quia nisi
      quos ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis
      odio sunt porro magni dolorum autem ipsa facere. Lorem ipsum dolor sit
      amet, consectetur adipisicing elit. Necessitatibus sed quia nisi quos
      ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis odio
      sunt porro magni dolorum autem ipsa facere.Lorem ipsum dolor sit amet,
      consectetur adipisicing elit. Necessitatibus sed quia nisi quos ipsum et
      possimus ducimus reiciendis deserunt quasi, alias facilis odio sunt porro
      magni dolorum autem ipsa facere.
    </p>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus
      sed quia nisi quos ipsum et possimus ducimus reiciendis deserunt quasi,
      alias facilis odio sunt porro magni dolorum autem ipsa facere.Lorem ipsum
      dolor sit amet, consectetur adipisicing elit. Necessitatibus sed quia nisi
      quos ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis
      odio sunt porro magni dolorum autem ipsa facere. Lorem ipsum dolor sit
      amet, consectetur adipisicing elit. Necessitatibus sed quia nisi quos
      ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis odio
      sunt porro magni dolorum autem ipsa facere.Lorem ipsum dolor sit amet,
      consectetur adipisicing elit. Necessitatibus sed quia nisi quos ipsum et
      possimus ducimus reiciendis deserunt quasi, alias facilis odio sunt porro
      magni dolorum autem ipsa facere.
    </p>
  </div>
);

export default Home;
