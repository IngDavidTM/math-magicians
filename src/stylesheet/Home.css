.home {
  display: flex;
  flex-direction: column;
  height: 80vh;
  align-items: center;
  justify-content: space-around;
  padding: 20px;
}

.home h2 {
  font-size: 26px;
  background-color: #ffffffc5;
  backdrop-filter: blur(5px);
  border-radius: 5px;
}

.home p {
  font-size: 20px;
  max-width: 1000px;
  text-align: justify;
  background-color: #ffffffc5;
  backdrop-filter: blur(5px);
  padding: 10px;
}
