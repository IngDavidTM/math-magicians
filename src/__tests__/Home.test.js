import renderer from 'react-test-renderer';
import Home from './pages/Home';
import { BrowserRouter } from 'react-router-dom';

it('should render Home page', () => {
  const renderHome = renderer
    .create(<BrowserRouter><Home /></BrowserRouter>)
    .toJSON();
  expect(renderHome.toMatchSnapshot();
})