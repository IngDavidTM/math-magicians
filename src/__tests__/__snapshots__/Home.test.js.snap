// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Home page 1`] = `
<div
  className="home"
>
  <h2>
    Welcome to our page
  </h2>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus sed quia nisi quos ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis odio sunt porro magni dolorum autem ipsa facere.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus sed quia nisi quos ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis odio sunt porro magni dolorum autem ipsa facere. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus sed quia nisi quos ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis odio sunt porro magni dolorum autem ipsa facere.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus sed quia nisi quos ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis odio sunt porro magni dolorum autem ipsa facere.
  </p>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus sed quia nisi quos ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis odio sunt porro magni dolorum autem ipsa facere.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus sed quia nisi quos ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis odio sunt porro magni dolorum autem ipsa facere. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus sed quia nisi quos ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis odio sunt porro magni dolorum autem ipsa facere.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus sed quia nisi quos ipsum et possimus ducimus reiciendis deserunt quasi, alias facilis odio sunt porro magni dolorum autem ipsa facere.
  </p>
</div>
`;
